src/styles/index.css

```diff
@@ -0,0 +1,94 @@
+@import "./stars.css";
```

Comment:
this is wrong file structure
in index you should gather all the file imports 
@import "./blocks/page.css";
@import "./blocks/product.css";
@import "./blocks/review.css";
@import "./blocks/stars.css";

and only link `index.css` to html